core/server/api_container/server/startosis_engine/kurtosis_instruction/tasks/run_sh.go

```diff
@@ -147,8 +154,21 @@ func (builtin *RunShCapabilities) Interpret(_ string, arguments *builtin_argumen
 		}
 	}
 
+	envVars := map[string]string{}
```

Comment:
i wonder whether we can extract this to tasks_shared.go

Reply:
Perfect, thanks for the hint!